app-modules/ai/src/Filament/Resources/AiAssistantResource/Pages/EditAiAssistant.php

```diff
@@ -106,6 +106,17 @@ public function getSaveFormAction(): Action
             });
     }
 
+    public static function canAccess(array $parameters = []): bool
+    {
+        $assistant = $parameters['record'];
```

Comment:
Let's add a docblock to type-hint this variable like `/** @var AiAssistant $assistant */